stock_reserve/test/stock_reserve.yml

```diff
@@ -6,7 +6,7 @@
     # Travis, the hook performing this operation for the new stock reservation
     # location is run after the test execution. This causes the stock level
     # computation to be wrong at the time the tests are run.
-    self._parent_store_compute(cr)
+    self._parent_store_compute()
```

Comment:
I use it in my v10 odoo, and it works. Is it only me who notice that stock_reserve in apps.odoo.com refers to different module than this one?